rakefile.netstd.rb

```diff
@@ -48,15 +49,13 @@
 ]
 
 repo = 'FakeItEasy/FakeItEasy'
-teamcity_server_url = 'http://teamcity.codebetter.com/admin/editBuildParams.html?id=buildType:bt929'
 appveyor_server_url = 'https://ci.appveyor.com/project/FakeItEasy/fakeiteasy/settings'

```

Comment:
This is the URL of the FakeItEasy Approval project; the FakeItEasy project is https://ci.appveyor.com/project/FakeItEasy/fakeiteasy-au76p (we might want to fix this, if possible - maybe by deleting FakeItEasy and renaming FakeItEasy Approval to FakeItEasy)